### ISSUE TITLE
Ships says destroyed, but does not disappear

### ISSUE THREAD

**Issue by nukeguard** [tsunamayo/Starship-EVO#2827]
### Short Description of the issue:

Fly any ship to a planet, ship hud will say destroyed, you can still interact with ship but it will no longer move or receive inputs. You can build on it and remove things, or even repair but still remains destroyed. I am not sure if this is planned feature not fully functioning or just that it's a bug.

### Steps to reproduce the issue:

need only fly a ship to a planet in lightcruise

### Environment
- Starship EVO version:  20w32b
- Graphics Card: gtx1080

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_
![image](https://user-images.githubusercontent.com/30876973/91523705-6d6bae00-e8cb-11ea-85b5-1065c4a6f355.png)

Please also include the `output_log.txt` file. You can find your log here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

**Comment by TIKIRobo** [comment]
Ships when destroyed are not supposed to disappear their wreckage just kinda sticks around , at least thats what I've seen, this will 
probably change in the future

**Comment by tsunamayo** [comment]
Yes not really a bug, in the experimental beta you will actually die and respawn your last medbay.
Destroyed ship does not disappear atm but the one destroyed in the planet should in the future.
Cheers
